### PR TITLE
Feat/read me

### PR DESCRIPTION
**Pull request by vatsal278**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,142 @@
...
+* Start the Docker container for mysql with command :
+```
+docker run --publish 9075:3306 -d mysql
+```
+
+* Start the Api locally with command :
+```
+go run .\cmd\TransactionManagementService\main.go
+```
+### You can test the api using post man, just import the [Postman Collection](./docs/transactionService.postman_collection.json) into your postman app.
```
Comment (PereRohit) [inline README.md:20]: Link not pointing to the collection. Check.

**Review by PereRohit — APPROVED** [review]
Parent branch should be the feat/get.

**Review by PereRohit — CHANGES_REQUESTED** [review]
Some testable scenarios are still available and can be added.
https://app.codecov.io/gh/vatsal278/TransactionManagementService/pull/4

File: README.md (hunk 1 above)
Comment (vatsal278) [reply README.md:20]: done

File: internal/router/router.go [hunk 2]
```diff
@@ -43,11 +43,12 @@ func attachTransactionManagementServiceRoutes(m *mux.Router, svcCfg *config.SvcC
 
 	router := m.PathPrefix("").Subrouter()
 	router.HandleFunc("", svc.NewTransaction).Methods(http.MethodPost)
-	router.HandleFunc("/download/{transaction_id}", svc.DownloadTransaction).Methods(http.MethodGet)
+
 	router.Use(middleware.ExtractUser)
 
 	router2 := m.PathPrefix("").Subrouter()
 	router2.HandleFunc("", svc.GetTransactions).Methods(http.MethodGet)
+	router2.HandleFunc("/download/{transaction_id}", svc.DownloadTransaction).Methods(http.MethodGet)
```
Comment (PereRohit) [inline internal/router/router.go]: This route shouldn't have cacher middleware.

**Review by PereRohit — CHANGES_REQUESTED** [review]

File: internal/router/router.go (hunk 2 above)
Comment (vatsal278) [reply internal/router/router.go]: done

**Review by PereRohit — APPROVED** [review]

**Pull request merged**